### PR TITLE
Updating logstash templates and kibana dashboards to work with ELK7

### PR DESCRIPTION
**Pull request by ellen-lau**
This pull request addresses the dashboards and ELK7 compatibility, and corrects the field naming in the logstash template.

### CODE REVIEW

**Comment by ellen-lau** [comment]
All the current updated dashboards should work correctly; missing the audit dashboard changes, still testing.

File: logstashCollector-1.0/Elastic7/18.0.0.3+/20181010/liberty_logstash_template_ibm.json [hunk 1]
```diff
@@ -8,38 +8,38 @@
...
+            "requestPort": { "type": "keyword", "index": "true" },
+            "remoteHost": { "type": "keyword", "index": "true" },
+            "requestMethod": { "type": "keyword", "index": "true" },
+            "requestProtocol": { "type": "keyword", "index": "true" },
+            "bytesReceived": { "type": "long", "index": "true" },
+            "elapsedTime": { "type": "long", "index": "true" },
+            "queryString": { "type": "text", "index": "true" },
+            "responseCode": { "type": "integer", "index": "true" },
+            "uriPath": { "type": "keyword", "index": "true" },
             "ibm_userAgent": { "type": "keyword", "index": "true" },
```
Comment (donbourne) [inline logstashCollector-1.0/Elastic7/18.0.0.3+/20181010/liberty_logstash_template_ibm.json]: I think ibm_userAgent should not have ibm_ prefix.

**Comment by mbuccarello** [comment]
Not sure about how these changes can help me to manage GC Events in the memory dashboard. What is unclear on memory dashboard why they expecting differnt index. Am I wrong?

**Comment by donbourne** [comment]
@mbuccarello , all of the events should be in the same index.  I'm wondering if what you are seeing is a result of the template you are using not including the fields that logstashCollector is sending.

**Review by donbourne — APPROVED** [review]

**Pull request merged**